pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -912,14 +909,9 @@
         %app
       :_  state
       :_  ~
-      :^  duct  %pass  /run-app-cancel/[app.action.u.connection]
-      ^-  note
-      :^  %m  %deal  [our our]  :-  app.action.u.connection
-      ::
-      ^-  task:agent:mall
-      :*  %poke
-          %handle-http-cancel
-          !>(inbound-request.u.connection)
+      :*  duct  %pass  /watch-response/[eyre-id]
+          %m  %deal  [our our]  app.action.u.connection
+          %leave  ~
```

Comment:
This probably doesn't matter since I assume there's no real usages of `handle-http-cancel`, but the application does get the ability to handle the leave event, right?

Reply:
Yes, the application can handle the `%leave` event, though it can't reply to it (which is why I can get rid of the section with your comment about explicitly ignoring the ack).